nominal/core/asset.py

```diff
@@ -229,6 +230,36 @@ def list_data_scopes(self) -> Sequence[tuple[str, ScopeType]]:
         """
         return (*self.list_datasets(), *self.list_connections(), *self._list_logsets(), *self.list_videos())
 
+    def get_or_create_dataset(
+        self,
+        *,
+        data_scope_name: str,
```

Comment:
```suggestion
        data_scope_name: str,
        *,
```
this seems fine as a positional arg